modules/virt-default-cluster-roles.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-additional-security-privileges-controller-and-launcher.adoc
+
+:_content-type: REFERENCE
+[id="default-cluster-roles-for-virt_{context}"]
+= Default cluster roles for {VirtProductName}
+
+By using cluster role aggregation, {VirtProductName} extends the default {product-title} cluster roles to include permissions for accessing virtualization objects.
+
+.{VirtProductName} cluster roles
+[cols="1,4",options="header"]
+|===
+
+|Default cluster role |Description
```

Comment:
This should be


|Default cluster role|{VirtProductName} Role | Description |
|-------------------------|--------------|------------|
| `view` | `kubevirt.io:view` | … |
| `edit` | `kubevirt.io:edit` | … |
| `admin` | `kubevirt.io:admin` | … |

Thus it would reflect which cluster role is extended by what kubevirt role

Reply:
Updated. :)